html/elements/script.json

```diff
@@ -0,0 +1,608 @@
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "1.0",
+                  "notes": "Starting in Firefox 4, inserting script elements that have been created by calling <code>document.createElement(\"script\")</code> no longer enforces execution in insertion order. This change lets Firefox properly abide by the specification. To make script-inserted external scripts execute in their insertion order, set <code>.async=false</code> on them."
+                },
+                "firefox_android": {
+                  "version_added": "1.0",
+                  "notes": "Starting in Firefox 4, inserting script elements that have been created by calling <code>document.createElement(\"script\")</code> no longer enforces execution in insertion order. This change lets Firefox properly abide by the specification. To make script-inserted external scripts execute in their insertion order, set <code>.async=false</code> on them."
+                },
+                "ie": {
+                  "version_added": true
+                },
+                "ie_mobile": {
+                  "version_added": true
+                },
+                "opera": {
+                  "version_added": true
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": true
+                },
+                "safari_ios": {
+                  "version_added": true
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          }
+        },
+        "async": {
+          "__compat": {
+            "basic_support": {
+              "support": {
+                "webview_android": {
+                  "version_added": true,
+                  "notes": "In older browsers that don't support the <code>async</code> attribute, parser-inserted scripts block the parser; script-inserted scripts execute asynchronously in IE and WebKit, but synchronously in Opera and pre-4.0 Firefox. In Firefox 4.0, the <code>async</code> DOM property defaults to <code>true</code> for script-created scripts, so the default behavior matches the behavior of IE and WebKit. To request script-inserted external scripts be executed in the insertion order in browsers where the <code>document.createElement(\"script\").async</code> evaluates to <code>true</code> (such as Firefox 4.0), set <code>.async=false</code> on the scripts you want to maintain order. Never call <code>document.write()</code> from an async script. In Gecko 1.9.2, calling <code>document.write()</code> has an unpredictable effect. In Gecko 2.0, calling <code>document.write()</code> from an async script has no effect (other than printing a warning to the error console)."
```

Comment:
I tried to divide it per browser, but the actual interest is the comparison between browser, so I removed these notes from here and add it them in the article (Fixing Gecko -> Firefox at the same time)

Fixed.